### ISSUE TITLE
Wrong string in release candidate branch using git-flow workflow

### ISSUE THREAD

**Issue by ilvetz** [ilvetz/z-bump#4]
On release branches, z-dump counts every commit from last master (not develop) for release candidate builds.

Example: expected build string on `H` is `a.b.c.1-rc1` but is `a.b.c.1-rc5`; expected build on `I` is `a.b.c.2` but is `0.1.0.1-rc5`.
```
master  - A--------------------I
           \                  / \
release ----\-------------G--H---\
             \           /        \
develop ------B------E--F----------J
               \    /
feature --------C--D
```

**Comment by ilvetz** [comment]
> expected build on `I`

is another issue at all. Git-flow does not execute pre-commit hook when merging `H` into master.

**Comment by ilvetz** [comment]
Fixed by [9ce8a64](https://github.com/ilvetz/z-bump/commit/9ce8a641c1714264e3ce9f4f92278acc59aa4289)
